db/seed.sql

```diff
@@ -7710,6 +7920,12 @@ INSERT INTO `eav_attribute_set` VALUES (5,5,'Default',1);
 INSERT INTO `eav_attribute_set` VALUES (6,6,'Default',1);
 INSERT INTO `eav_attribute_set` VALUES (7,7,'Default',1);
 INSERT INTO `eav_attribute_set` VALUES (8,8,'Default',1);
+INSERT INTO `eav_attribute_set` VALUES (9,4,'Books',0);
```

Comment:
Each category of product now has its own attribute set.  This will make management in the future much easier.